pkg/server/server.go

```diff
@@ -64,6 +64,10 @@ type jsGlobals struct {
 	CustomLogoURL            string `json:"customLogoURL"`
 	StatuspageID             string `json:"statuspageID"`
 	DocumentationBaseURL     string `json:"documentationBaseURL"`
+	AlertManagerPublicURL    string `json:"alertManagerPublicURL"`
```

Comment:
"Alertmanager" is written as one word, so we should use `AlertmanagerPublicURL` and `alertmanagerPublicURL`.

Reply:
Was following the pattern we already use https://github.com/openshift/console/blob/master/pkg/server/server.go#L60
Would change this name nit globally as a followUP